colcon_cmake/task/cmake/__init__.py

```diff
@@ -65,6 +65,11 @@ async def has_target(path, target):
         "'has_target' not implemented for CMake generator '{generator}'" \
         .format_map(locals())
 
+async def get_makefile_has_targets(path, target):
```

Comment:
This needs a docstring, similar to the one for `get_makefile_targets()`.

Reply:
added